target/linux/ramips/dts/Newifi-D2.dts

```diff
@@ -0,0 +1,155 @@
+/dts-v1/;
+
+#include "mt7621.dtsi"
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+/ {
+	compatible = "pandorabox,newifi-d2", "mediatek,mt7621-soc";
+	model = "Newifi-D2";
+
+	memory@0 {
+		device_type = "memory";
+		reg = <0x0 0x10000000>;
+	};
+
+	chosen {
+		bootargs = "console=ttyS0,115200";
+	};
+
+	gpio-leds {
+		compatible = "gpio-leds";
+
+		power {
+			label = "newifi-d2:amber:power";
+			gpios = <&gpio0 6 GPIO_ACTIVE_LOW>;
+		};
+
+		sys {
```

Comment:
Could it be that it is the usb led? If so, rename the LED and use set_usb_led() in 01_leds.

Reply:
No. The power led and the following sys led are two LEDs with different color but at the same place. There is another blue USB led using GPIO10.

Reply:
Please add the missing usb led to the dts and add the usb led trigger using set_usb_led().

Reply:
Why do you enable the sys LED by default? (`ucidef_set_led_default "sys" "sys" "$board:blue:sys" "1"`)